src/objects/overview/overview.scss

```diff
@@ -0,0 +1,56 @@
+@use "../../compiled/tokens/scss/breakpoint";
+@use '../../mixins/ms';
+@use "../../compiled/tokens/scss/size";
+@use '../../mixins/fluid';
+
+/**
+ * The Overview stacks content vertically on small screens.
+ * On large screens the header and actions are stacked horizontally above the content.
+ * The specific spacing values were inferred from the Article Listing prototype.
+ */
+
+.o-overview__header {
+  margin-bottom: ms.step(1);
+}
+
+.o-overview__actions {
+  margin-bottom: ms.step(3);
+}
+
+@supports (display: grid) {
```

Comment:
Do you think this is necessary? We aren't optimizing for IE, and the content seems perfectly accessible without grid if the spacing's a little off in this section.

I'm open to it, I was just surprised to see these fallbacks and then the margin resets later on for a relatively simple layout object.

Reply:
I don't have strong feelings here. 

The only change for IE11 was the `@supports` statement itself. The margins and margin resets were to handle the layout changes between small screens and large screens. I attempted to simplify this an use `display: grid` with gaps on all screen sizes, but this didn't quite match the prototype design. (On small screens there are two different margin sizes, so it doesn't map to a consistent gap.)

All that said, I'm totally fine removing the `supports` statement if it's just extra cruft.

Reply:
I didn't realize the margin resets were also a responsive concern. In that case this isn't much cruft at all.